src/api/app/assets/javascripts/webui/delete_confirmation_dialog.js

```diff
@@ -1,4 +1,18 @@
-function setValuesOnDeleteConfirmationDialog(modalId) { // jshint ignore:line
+function collectDeleteConfirmationModalsAndSetValues() { // jshint ignore:line
+  $.each(modalIds(), function( _index, modalId ) {
+    setValuesOnDeleteConfirmationDialog(modalId);
+  });
+}
+
+function modalIds() {
+  var targets = $('a[data-toggle="modal"][data-target^="#delete"]').toArray().map(
```

Comment:
Is this `toArray` really needed?

Reply:
Yes, if I don't use `toArray` here, I'll get an Object as result, instead of an array of ids.

This might be handled differently, but for me it was simpler to use `Array.map()` as explained [here](https://www.geeksforgeeks.org/javascript-array-map-method/).

Reply:
Okay